improver_tests/utilities/test_CalculateClimateAnomalies.py

```diff
@@ -273,9 +281,9 @@ def test_error_units_mismatch(diagnostic_cube, mean_cube, std_cube, error_to_che
         ):
             plugin.verify_units_match(diagnostic_cube, mean_cube, std_cube=None)
     else:
-        std_cube.units = "C"  # The units should be K ordinarily
+        std_cube.units = "C"  # The units should be 1 (dimensionless) ordinarily
         with pytest.raises(
-            ValueError, match="The standard deviation cube must have the same units "
+            ValueError, match="The standard deviation cube must be dimensionless"
```

Comment:
```suggestion
            ValueError, match="The standard deviation cube have the same units"
```
This was correct before.